submissions/AndriiZahumnyi/js-dom/script/index.js

```diff
@@ -0,0 +1,33 @@
+
+/* Toggle menu */
+
+const hamburgerToggle = document.querySelector('#menu-toggle');
+
+hamburgerToggle.addEventListener('click', function (e) {
+  
+  e.preventDefault();
+
+  document.querySelector('.flexmenu').classList.toggle('active');
+});
+
+/* Aside menu -  interactive side-menu*/
+
+const arrAside = ["aside-html", "aside-css", "aside-bootstrap", "aside-js", "aside-jquery", "aside-git"];
+const arrContent = ["html-section", "css-section", "bootstrap-section", "js-section", "jquery-section", "git-section"];
+
+const myCustomDiv = document.getElementById("aside-container");
```

Comment:
I'd recommend you to rename this variable name. It's not so clear now. 
You can create a whole bunch of custom divs and who knows which of the divs would be exact div to work with.